packages/cms/src/components/Viz/PercentageBar.jsx

```diff
@@ -55,49 +53,83 @@ class PercentageBar extends Component {
       this.setState({config: null}, this.buildConfig.bind(this));
     }
   }
-
   render() {
     const {showAll, config} = this.state;
-
     if (!config) return null;
-
-    const {data, cutoff, cutoffText, title, value, groupBy, sort, total, numberFormat, showText, hideText} = config;
-
+    const {data, cutoff, cutoffText, title, value, groupBy, sort, total, numberFormat, isPercentValues, showText, hideText} = config;
     const cutoffFunction = typeof cutoff === "number" ? data => data.slice(0, cutoff) : cutoff;
-
     let displayData = showAll ? data : cutoffFunction(data);
 
     if (sort) displayData = displayData.sort(sort);
 
+    // Create ticks and labels
+    let labelVal, xPos;
+    let lines = [];
+    let ticks = [];
+    for (let i = 0; i <= 10; i++) {
+      // ensure first line isn't cropped left
+      xPos = i === 0 ? '0.1%' : i*10 + '%';
+
+      // Determine whether to display labels as percentages
+      if(isPercentValues){
+        labelVal = i === 0 ? '0%' : Math.round(total*(i*0.1)) + '%';
+      }else{
+        labelVal = i === 0 ? 0 : (total*(i*0.1)).toFixed(2);
+      }
+
+      lines.push(<line key={i} x1={xPos} x2={xPos} y1="0" y2="100%"/>);
+      ticks.push(<text key={i} x={xPos} y="100%">{labelVal}</text>);
+    }
+
     return (
-      <div className="percentage-bar-wrapper">
+      <>
```

Comment:
what's this empty element that's been added?

Reply:
that's the shorthand for `<React.Fragment>`